### DISCUSSION TITLE
Tillæg til spotpris

### DISCUSSION THREAD

**Discussion by ClausSamuelsen** [rndfm/elspotpris#177] [General]
I leverandøroversigten har "Vindstød Danskvind" et tillæg på 0,005 kr/kwh
Men da jeg ville skifte leverandør i dag skriver Vindstød: "Du betaler spotprisen tillagt kun 0,63 øre pr. kWh".
Det er jo meget forskelligt fra det, der fremgår af leverandøroversigten.
Så spørsgmålet er om priserne i elspotpriser ikke er opdateret længe, eller er der noget jeg har misforstået?
mvh
Claus

**Comment by rndfm** [comment]
Vindstød står pt. med et tillæg på 0,005 kr/kwh eksl. moms. (ved ikke om du har slået moms fra på https://elspotpris.dk/live ?)

På deres hjemmeside vises der 0,63 øre/kwh inkl. moms.

0,005 kr plus moms = 0,00625. Rundet up til nærmeste hele øre er det 0,0063 kr. Eller 0,63 øre.

**Reply by ClausSamuelsen** [reply]
Tak, jeg kan åbenbart ikke kende forskel på kroner og ører :(
